orchagent/portsorch.cpp

```diff
@@ -917,6 +917,36 @@ bool PortsOrch::setPortMtu(sai_object_id_t id, sai_uint32_t mtu)
     return true;
 }
 
+
+bool PortsOrch::setPortTpid(sai_object_id_t id, sai_uint16_t tpid)
```

Comment:
Can we combine this with Lag to have single function and pass `is_lag `as boolean since only two lines are different and rest are the same? Just a suggestion.

Reply:
There is a chance that we may end up having to implement the LAG handling by having Orchagent apply the LAG TPID config to all its LAG members.  This is because currently only one SAI vendor supports TPID setting on Port only and not yet for LAG.  For our fanout switch usage, this is already good enough.  But if we ever want to apply it to LAG while SAI vendor not supporting it, it will be much easier to have the port/LAG handling having its own function to do the handling.  So for now, let's keep the way it is just in case the enhancement is needed in the future.

Reply:
ok